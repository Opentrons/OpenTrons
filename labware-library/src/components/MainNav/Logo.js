// @flow
// top nav bar logo image
import * as React from 'react'
import logo from './images/ot-logo-full.png'

import styles from './MainNav.module.css'

export function Logo(): React.Node {
  return (
    <a href="https://www.opentrons.com/">
      <img className={styles.logo} src={logo} />
    </a>
  )
}
