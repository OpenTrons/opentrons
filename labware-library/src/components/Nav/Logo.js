// @flow
// top nav bar logo image
import * as React from 'react'

import logoSrc from './images/ot-logo-full.png'
import styles from './styles.css'

export default function Nav () {
  return <img className={styles.logo} src={logoSrc} />
}
