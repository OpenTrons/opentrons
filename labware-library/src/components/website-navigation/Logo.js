// @flow
// top nav bar logo image
import * as React from 'react'

import logoSrc from './images/ot-logo-full.png'
import styles from './styles.css'

export default function Logo() {
  return <img className={styles.logo} src={logoSrc} />
}
