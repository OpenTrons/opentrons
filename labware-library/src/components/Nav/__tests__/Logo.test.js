// @flow
// tests for Logo image component
import * as React from 'react'
import Renderer from 'react-test-renderer'

import Logo from '../Logo'

describe('Logo', () => {
  test('component renders', () => {
    const tree = Renderer.create(<Logo />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders an <img>', () => {
    const tree = Renderer.create(<Logo />)
    // throws if can't find 'img'
    tree.root.findByType('img')
  })
})
