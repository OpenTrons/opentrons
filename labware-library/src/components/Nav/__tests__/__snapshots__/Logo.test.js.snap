// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders 1`] = `
<img
  className="logo"
  src="ot-logo-full.png"
/>
`;
